src/__stories__/custom-input-story.tsx

```diff
@@ -0,0 +1,269 @@
+import * as React from 'react';
```

Comment:
I find this annoying
https://user-images.githubusercontent.com/579417/149164205-cc34a187-9149-4b69-aa49-212fcbbf756a.mp4
Not sure if it is easy to fix. If you click outside the menu to close it, change the tab and go back to it, the menu opens automatically